app/org/sagebionetworks/bridge/json/BridgeObjectMapper.java

```diff
@@ -54,12 +55,24 @@ public void setupModule(SetupContext context) {
                 context.addBeanSerializerModifier(bsm);
             }
         });
-        SimpleModule serializers = new SimpleModule();
-        serializers.addDeserializer(Schedule.class, new ScheduleDeserializer());
-        serializers.addSerializer(Schedule.class, new ScheduleSerializer());
-        this.registerModule(serializers);
-        
+        this.registerModule(new JodaModule());
         this.registerModule(new LowercaseEnumModule());
+        this.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
+        /* This works, but we don't want to do it. While it's nice to have shorter dates and times in some classes, like 
+         * the Schedule (where seconds/milliseconds aren't needed), we do not know if this creates challenges for some 
+         * language libraries when parsing ISO 8601 strings, and there are DateTimes in the system that we do want to 
+         * server as timestamps to the millisecond. If used, we'd need to scope to specific properties of specific objects.
```

Comment:
Let's remove this comment and the commented out code. We already have examples of custom Joda Jackson serializers in our JSON package, and if we really want to serialize LocalTimes in just HH:mm format, we can write a custom serializer for that and use the JsonSerialize annotation.